lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/sim/XwikiBridgeInterfaceRestResource.java

```diff
@@ -119,8 +119,28 @@ public void updateConfiguration(String coreFacadeHostname, int coreFacadeHostPor
 	
 	@Override
 	public ProcessData getProcessInfos(String processArtifactId) {
-		// TODO Auto-generated method stub
-		return null;
+		HttpClient httpClient = RestResource.getAnonymousClient();
+		String uri = String.format("%s/learnpad/sim/bridge/processes/%s",
+				RestResource.SIM_REST_URI,processArtifactId);
+
+		GetMethod getMethod = new GetMethod(uri);
+		getMethod.addRequestHeader("Content-Type", "application/json");
+		
+		ProcessData unmashelledProcessData = new ProcessData();
+		
+		try {
+
+			httpClient.executeMethod(getMethod);
+		    
+			ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
@tomjorquera I implemented the body of the methods in that manner following the "style" we adopted for all the *RestResource.java classes. But  I may accept your suggestion.